conda_smithy/cli.py

```diff
@@ -52,6 +53,23 @@ def generate_feedstock_content(target_directory, source_recipe_dir):
         with feedstock_io.write_file(forge_yml) as fh:
             fh.write("{}")
 
+    # merge in the existing configuration in the source recipe directory
+    forge_yml_recipe = os.path.join(source_recipe_dir, "conda-forge.yml")
+    yaml = YAML()
+    if os.path.exists(forge_yml_recipe):
+        os.unlink(os.path.join(target_recipe_dir, "conda-forge.yml"))
+        try:
+            with open(forge_yml_recipe, 'r') as fp:
+                _cfg = yaml.load(fp.read())
+        except:
+            _cfg = {}
+
+        with open(forge_yml, 'r') as fp:
+            _cfg_feedstock = yaml.load(fp.read())
+            _cfg_feedstock.update(_cfg)
```

Comment:
I think we want a recursive update here to allow us to replaces bits multiply nested keys.

Reply:
yeah, i guess, though atm in almost all cases this is {}